cypress/integration/scheduled-events-table.spec.ts

```diff
@@ -204,26 +204,24 @@ context('Scheduled events table tests', () => {
 
       cy.contains(newScheduledEvent1.endsAt)
         .parent()
-        .find('[title="View event"]')
+        .find('[data-testid="VisibilityIcon"]')
```

Comment:
And here

Reply:
I think I can put our own data-cy but as long as this one exists not sure if it is needed.